collections/_pages/newcomers.html

```diff
@@ -0,0 +1,84 @@
+---
+layout: default
+#title: "Newcomers | Layer5"
+permalink: /newcomers
+redirect_from: newcomers/
+excerpt: "Newcomers | Layer5"
+description: "Contributions from Newcomers are actively encouraged in Layer5. Find a Meshmate and start contributing!"
+---
+
+<div class="container">
+    <div id="newcomers" class="section scrollspy">
+        <h3 style="text-align: center; color: aliceblue;">
+            <div class="white-text">Newcomers at Layer5</div>
+        </h3>
+        <div class="row white-bg row-view">
+            <div class="col s12 m12 col-view">
+                <img src="/assets/images/welcome_newcomers.svg" width="35%" align="right">
+                {% include newcomers/newcomers.html %}
+            </div>
+        </div>
+        <div class="row white-bg row-view">
+            <div class="col s12 m12 col-view">
+                <h3 class="center">Tutorials</h3>
+                <table style="margin:0px; padding:0px;width:100%">
+                    <th>Date</th>
+                    <th>Topic</th>
+                    <th>Resources</th>
+                    <th>Presenters</th>
+                    <tr>
+                        <td>Oct. 1st, 2020</td>
+                        <td><i>An Introduction to Contributing to Meshery</i></td>
+                        <td>
+                            <a
+                                href="https://docs.google.com/presentation/d/1Wc5ALdn-G3fADJ8I6nJlyOGhV2XHVprYPqDvkZ1MEqY/edit#">slides</a>,<a
+                                href="https://youtu.be/67iy2JEp4Ss">recording</a>
+                        </td>
+                        <td>
+                            <a href="/community/members/vijay-cherukuri">Vijay Cherukuri</a>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td>Oct. 8th, 2020</td>
+                        <td><i>An Introduction to mesheryctl</i></td>
+                        <td><a
+                                href="https://docs.google.com/presentation/d/1LxhzJhUs9-Hc9mwlKVUMlDAVdH-QWLvU1KWxVuRbSHg/edit?usp=sharing">slides</a>,<a
+                                href="https://youtu.be/wK7Q-zbJ3gQ?t=3">recording</a></td>
+                        <td><a href="/community/members/nupur-thakur">Nupur Thakur</a></td>
+                    </tr>
+                    <tr>
+                        <td>Oct. 15th, 2020</td>
+                        <td><i>A tutorial on contributing to Layer5 and working with Git </i></td>
+                        <td><a
+                                href="https://docs.google.com/presentation/d/1UOlwFtZ-VJhW4RgQoSI_QOGUQ7OBG8NZDWrFUor7eyY/edit?usp=sharing">slides</a>,<a
+                                href="https://youtu.be/O9UZO5g9BvI">recording</a></td>
+                        <td><a href="https://layer5.io/community/members/ruth-ikegah">Ruth Ikegah</a></td>
```

Comment:
Make the links uniform as above.

Reply:
@Jashpatel1 What do you mean uniform?

Reply:
change the links to /community/members/ruth-ikegah

Reply:
You mean point it to the directory and not the web address?